packages/hebao_v1/contracts/modules/core/ERC1271Module.sol

```diff
@@ -8,14 +8,14 @@ import "../../lib/AddressUtil.sol";
 import "../../lib/ERC1271.sol";
 import "../../lib/SignatureUtil.sol";
 import "../../thirdparty/BytesUtil.sol";
-import "../base/BaseModule.sol";
+import "../security/SecurityModule.sol";
 
 
 /// @title ERC1271Module
 /// @dev This module enables our smart wallets to message signers.
 /// @author Brecht Devos - <brecht@loopring.org>
 /// @author Daniel Wang - <daniel@loopring.org>
-abstract contract ERC1271Module is ERC1271, BaseModule
+abstract contract ERC1271Module is ERC1271, SecurityModule
```

Comment:
It's a bit wierd that this module is also a SecurityModule...

Reply:
Was only done to fix compile problems. Otherwise `FinalSecurityModule` inherits from both `BaseModule` here and `SecurityModule` in `ForwarderModule` which needs reimplementation from certain virtual functions (multiple inheritance problems, solidity specific).